src/pages/article/birds-are-friends-too/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: A gentle introduction to birdwatching
+date: '2019-05-30T17:37:26.785Z'
```

Comment:
probably change this to `date: '2019-05-19T17:37:26.785Z'` so it is for today's date